client/flutter/assets/content_bundles/symptom_checker.en.yaml

```diff
@@ -0,0 +1,21 @@
+schema_version: 1
+content_version: 1
+contents:
+  type: symptom_checker
+  items:
+    -
+      id:  question_1
```

Comment:
@brunobowden Not sure you got a chance to look at the full proposal (the PR just has a minimal test) but I think we can cover what you described above easily with it:  https://docs.google.com/document/d/1c7IvXZ2JNCt2vb1cVr5M2Gx5V8r3cU2E-PplTcsPxtY/edit#heading=h.b1psbq4oi4iu

We discussed the possibility of implementing the full SurveyJS format but it seems like it is way more complicated than what we need. We also talked about supporting a subset of SurveyJS, but that seems like it would be the worst of both worlds - incompatibility without the flexibility to tailor for things we use.